### ISSUE TITLE
Support Django 1.5

### ISSUE THREAD

**Issue by ghing** [denverfoundation/storybase#660]
[Django 1.5 was released](https://www.djangoproject.com/weblog/2013/feb/26/15/) yesterday.  According to [this blog comment](https://www.django-cms.org/en/blog/2013/01/15/24-beta-released/#comment-773770291), the next release of Django CMS won't yet support Django 1.5, but I should start testing to identify any issues early on.

**Comment by ghing** [comment]
I'd say this is lower priority than supporting the new version of Tastypie (see #615).

**Comment by ghing** [comment]
Django CMS now supports Django 1.5. Also, the latest stable release of Tastypie, 0.10 requires Django 1.5

**Comment by ghing** [comment]
This is a list of changes for upgrading support of Django 1.5 and Django CMS 2.4.*: http://docs.django-cms.org/en/2.4.0/upgrade/2.4.html

It looks like some of the internationalization issues will affect us

**Comment by ghing** [comment]
Also see the [release notes for Django 1.5](https://docs.djangoproject.com/en/dev/releases/1.5/)

**Comment by jwirfs-brock** [comment]
@ghing Should I repeat the tests I did in Firefox for Chrome and Safari?

**Comment by ghing** [comment]
@jwirfs-brock Don't worry about that as there's not much client-side stuff.

**Comment by jwirfs-brock** [comment]
@ghing Ok. Are there any admin tests you can think of that I missed?

**Comment by ghing** [comment]
@jwirfs-brock I'll try and think of other testing tasks and get the instances in a state where you can test in parallel to my development after I get a little farther on the text editor auto-resize issue.
